tests/main/interfaces-openvswitch/task.yaml

```diff
@@ -15,10 +17,13 @@ details: |
     list and deletion.
 
 prepare: |
-    echo "Given openvswitch is installed"
+    # Note that the order of the installs is to avoid a timeout issue on the 
```

Comment:
Curious - the order has not changed in this PR? Is this comment still valid? Or am I missing something?